MdePkg/Library/BaseFdtLib/FdtLib.c

```diff
@@ -205,6 +206,45 @@ FdtNextSubnode (
   return fdt_next_subnode (Fdt, Offset);
 }
 
+/**
+  Returns a offset of next node from the given node.
+
+  @param[in] Fdt            The pointer to FDT blob.
+  @param[in] Offset         The offset to previous node.
+
+  @return The offset to next node offset.
+
+**/
+INTN
+EFIAPI
+FdtNumRsv (

```

Comment:
could we highlight this is "memory reserve map" for the description and function naming?
MdePkg\Library\BaseFdtLib\libfdt\libfdt\libfdt.h:
 * fdt_num_mem_rsv - retrieve the number of memory reserve map entries

Reply:
Done.

Reply:
resolved